lib/high_voltage/page_collector.rb

```diff
@@ -0,0 +1,47 @@
+module HighVoltage
+
+  require 'find'
+
+  class PageCollector
+
+    attr_reader :page_ids, :content_path
+
+    def initialize(content_path)
+      @content_path = content_path.to_s
+    end
+
+    def page_ids
+      page_ids = []
+
+      Find.find(content_path) do |f|
+        page_ids.push url_path(f) if !directory?(f) && html?(f) && !partial?(f)
+      end
+
+      page_ids
+    end
+
+    private
+
+    def url_path(f)
+      f.gsub(content_path, '').gsub(file_pattern, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.